src/System.IO.FileSystem/tests/File/GetSetTimes.cs

```diff
@@ -92,5 +92,20 @@ public void PageFileHasTimes()
                 });
             }
         }
+
+        [Fact]
+        public void SetLastWriteTimeMilliSec()
+        {
+            string firstFile = GetTestFilePath();
+            string secondFile = GetTestFilePath();
+
+            File.WriteAllText(firstFile, "");
+            File.WriteAllText(secondFile, "");
+
+            File.SetLastAccessTimeUtc(secondFile, DateTime.UtcNow);
+            long firstFileTicks = File.GetLastWriteTimeUtc(firstFile).Ticks;
```

Comment:
@Anipik since we're now bringing a second fix to milliseconds timestamps to ship room they will all whether we have checked the test of the BCL to make sure there isn't a third. If you haven't already I would look at all places we directly handle file time stamps in our native and managed code to make sure we are done with these problems.

Reply:
I have taken a look at other places. But i will test all the properties independently for millisec/nanosecond to be sure on unix systems.

Reply:
@danmosemsft [update] there are still some places\codePaths where we need to correct things. I will put up the PR soon for that

Reply:
Great, lets get all ducks in a row and PR(s) in master etc before involving shiproom